persistent-template/Database/Persist/TH.hs

```diff
@@ -437,6 +439,24 @@ data MkPersistSettings = MkPersistSettings
     -- False.
     , mpsPrefixFields :: Bool
     -- ^ Prefix field names with the model name. Default: True.
+    --
+    -- Note: this field is deprecated. Use the mpsFieldLabelModifier  and mpsConstraintLabelModifier instead.
```

Comment:
We can use a [`DEPRECATED` pragma](https://downloads.haskell.org/~ghc/7.2.1/docs/html/users_guide/pragmas.html#warning-deprecated-pragma) to make this official.